code/src/App.js

```diff
@@ -1,12 +1,26 @@
 import React from 'react'
 import data from './data.json'
+import { Album } from './components/Album'
+import { Header } from './components/Header'
+import './App.css'
 
 console.log(data)
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
+    <div className="albumComponent">
+      <div className="headerSection">
+        <Header />
+      </div>
+      {data.albums.items.map((item) => (
+        <Album
+          key={item.id}
+          albumTitle={item.name}
+          urlArtistName={item.artists[0].external_urls.spotify}
+          image={item.images[1].url}
+          artistName={item.artists[0].name}
```

Comment:
Regarding the artists.
Since some of the albums have multiple artists you should make sure to show all of them.
Pass along the whole artists-array as a prop, so instead of urlArtistName and artistName go for artists={album.artists}